stdlib/public/Distributed/LocalTestingDistributedActorSystem.swift

```diff
@@ -27,33 +26,25 @@ import WinSDK
 /// for learning about `distributed actor` isolation, as well as early
 /// prototyping stages of development where a real system is not necessary yet.
 @available(SwiftStdlib 5.7, *)
-public struct LocalTestingDistributedActorSystem: DistributedActorSystem, @unchecked Sendable {
+public final class LocalTestingDistributedActorSystem: DistributedActorSystem, @unchecked Sendable {
```

Comment:
This UNDOES the workaround (which actually did not work after all).

This is NOT a change in the API of the system